### ISSUE TITLE
[librsvg] Unable to build

### ISSUE THREAD

**Issue by uberhacker** [skycocker/chromebrew#4503]
```
   Compiling librsvg_c_api v0.0.1 (/usr/local/tmp/crew/librsvg-2.50.1.tar.xz.dir/librsvg-2.50.1/librsvg)
    Finished release [optimized + debuginfo] target(s) in 11m 18s
  CCLD     librsvg-2.la
  CCLD     rsvg-convert
  GISCAN   Rsvg-2.0.gir
  GICOMP   Rsvg-2.0.gir
 VAPIGEN librsvg-2.0.vapi
Rsvg-2.0.gir:1875.5-1875.27: error: unknown child element `docsection' in `namespace'
Rsvg-2.0.gir:2058.5-2058.27: error: unknown child element `docsection' in `namespace'
GModule-2.0.gir:377.5-377.19: error: unknown child element `docsection' in `namespace'
make[2]: *** [/usr/local/share/vala/Makefile.vapigen:61: librsvg-2.0.vapi] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/usr/local/tmp/crew/librsvg-2.50.1.tar.xz.dir/librsvg-2.50.1'
make[1]: *** [Makefile:1277: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/librsvg-2.50.1.tar.xz.dir/librsvg-2.50.1'
make: *** [Makefile:828: all] Error 2
librsvg failed to build: `make V=0 -j4` exited with 2
```

**Comment by uberhacker** [comment]
Also on x86_64:
```
configure: error: vapigen >= 0.17.1.26 not found
librsvg failed to build: `./configure --prefix=/usr/local --libdir=/usr/local/lib64 --mandir=/usr/local/share/man --build=x86_64-cros-linux-gnu --host=x86_64-cros-linux-gnu --target=x86_64-cros-linux-gnu --enable-introspection=yes --enable-vala=yes --disable-static --enable-pixbuf-loader --disable-tools` exited with 1
```

**Comment by yledoare** [comment]
Update vala

**Comment by uberhacker** [comment]
Fixed in #4550.
